### PR TITLE
Added a new Dataset in translate directory

### PR DESCRIPTION
**Pull request by praveenjune17**
I have collected 370,000 lines of English-Tamil parallel text for training Machine translation models, I would like to contribute this to tfds. Data has been collected from three sources mentioned below

URL 1 of dataset: http://ufal.mff.cuni.cz/~ramasamy/parallel/html/
URL 2 of dataset: http://opus.nlpl.eu/
URL 3 of dataset: https://github.com/joshua-decoder/indian-parallel-corpora
Licenses of the datasets: CC , Unknown and CC

### CODE REVIEW

File: tensorflow_datasets/translate/en_tam_parallel_text.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+
+_CITATION = """
+@inproceedings {biblio:RaBoMorphologicalProcessing2012,
```
Comment (us) [inline tensorflow_datasets/translate/en_tam_parallel_text.py]: Please use same indentation for citations.

File: tensorflow_datasets/translate/en_tam_parallel_text.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            num_shards=1,
+            gen_kwargs={
+                'data_file': os.path.join(extracted_path_train,\
+				          'en_tam_parallel_corpus_train')}),
+        tfds.core.SplitGenerator(
+            name=tfds.Split.VALIDATION,
+            num_shards=1,
+            gen_kwargs={\
```
Comment (us) [inline tensorflow_datasets/translate/en_tam_parallel_text.py]: Unnecessary \ usage for newline

File: tensorflow_datasets/translate/en_tam_parallel_text.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+datasets were created from extracting text from movie subtitles, Bible, Quran, cinema and news websites so the quality of translations are of mixed.
+However, it should be useful enough to get you started training models.
+
+Reference for dataset 1 :- http://ufal.mff.cuni.cz/~ramasamy/parallel/html/
+Reference for dataset 2 :- http://opus.nlpl.eu/
+Reference for dataset 3 :- https://github.com/joshua-decoder/indian-parallel-corpora
+License   for dataset 3 :- https://github.com/joshua-decoder/indian-parallel-corpora/blob/master/LICENSE
+"""
+test_file_link = 'https://github.com/praveenjune17/Neural-Machine-Translation-English-Tamil-model/raw/master/en_tam_parallel_text_test.tar.gz'
+valid_file_link = 'https://github.com/praveenjune17/Neural-Machine-Translation-English-Tamil-model/raw/master/en_tam_parallel_text_valid.tar.gz'
```
Comment (us) [inline tensorflow_datasets/translate/en_tam_parallel_text.py]: Has the dataset any original test and valid files? Thanks for thinking and adding but you must need to give original files links.

**Comment by us** [comment]
Thanks for contributing! @praveenjune17

File: tensorflow_datasets/translate/en_tam_parallel_text.py (hunk 3 above)
Comment (praveenjune17) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: Except the 3rd dataset (joshua-decoder) the other datasets didn't have any original test or valid files. Mentioned the original file links in "en_tam_parallel_text.py"( in  the description) , I collected the data from these sources before I even know about TFDS and stored in my local PC. Even after doing some preprocessing to the best of knowledge there were many manual corrections done in the dataset to keep the noise low.  If the original files links are still needed let me know I'll figure out a way to do it.

Thanks !

File: tensorflow_datasets/translate/en_tam_parallel_text.py (hunk 1 above)
Comment (praveenjune17) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: Made changes . Please verify

File: tensorflow_datasets/translate/en_tam_parallel_text.py (hunk 2 above)
Comment (praveenjune17) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: Made changes . Please verify

**Comment by praveenjune17** [comment]
Please review the changes

File: tensorflow_datasets/translate/en_tam_parallel_text.py (hunk 3 above)
Comment (us) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: Yes, but the dataset has to be the original link for tfds. 
@rsepassi @Conchylicultor @cyfra what do you think about that?

Comment (praveenjune17) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: https://github.com/praveenjune17/Neural-Machine-Translation-English-Tamil-model/blob/master/Transformer_en_tam.ipynb

Please have a look at the above machine translation script. I have patched the en_tam scripts into the /usr/local.../translate directory and trained the transformer by loading the data using tfds. It works like a gem , the translation output is also good. As I already mentioned , I have done many manual corrections in the dataset to keep the noise low and I'm not sure whether I would get the same quality of translations without those manual corrections .

Comment (us) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: Yes, but it is better if you download the files from the original source and make corrections to the tfds dataset file.

Comment (praveenjune17) [reply tensorflow_datasets/translate/en_tam_parallel_text.py]: sure..started working on that

**Pull request closed without merging**